docs/code_overview/other.rst

```diff
@@ -39,6 +39,7 @@ them bound to an attribute of one of the PRAW models.
 
     other/commentmoderation
     other/fullnamemixin
+    other/inboxablemixin
     other/submissionmoderation
     other/rulemoderation
```

Comment:
```suggestion
    other/inboxablemixin
    other/rulemoderation
    other/submissionmoderation
```
Wold you mind sorting this while we're at it?

Reply:
Good catch. I believe both are fixed in the rebase.